packages/web-components/.storybook/container.scss

```diff
@@ -40,6 +40,8 @@ body,
 [name='main-content'] {
   position: relative;
   height: 100%;
+  overflow-y: scroll;
+  scroll-behavior: smooth;
```

Comment:
Adding this to all stories seems to have done some weird things across all stories, can this be set only for the story that needs it?

Reply:
@jeffchew what weird things are you seeing out of curiosity? If it adds weird things, would that create weird things happening when someone builds a page with Table of contents?

Reply:
Hey @photodow it might be more related to how the headless browser sets up the snapshots. The configuration only allows setting width and not height, and chrome driver specifically could be setting the frame with a shorter height as a side effect with the outer body element being set to overflow. So it might be a limitation of the testing suite, or maybe it's surfacing another issue.

Reply:
@jeffchew this problem was fixed, and now the changes Percy's pointing are intended, as per @photodow request.